Script.js

```diff
@@ -215,4 +215,49 @@ workSection.addEventListener('click', (event) => {
     const getProjectId = event.target.parentElement.parentElement.id;
     popupWindow(getProjectId);
   }
+});
+
+//  Form Validation
+const form = document.querySelector('#contact_section form');
+const email = form.querySelector('#email');
+const error = form.querySelector('.error');
+const submitButton = form.querySelector('#form_submit');
+const senderMessage = form.querySelector('#msg');
+
+function toggleValidationClasses(input) {
+  input.classList.toggle('error-input');
+  error.classList.toggle('hide');
+}
+
+function setError(element, message) {
+  error.innerText = message;
+  toggleValidationClasses(element);
+}
+
+function setSuccess() {
+  senderMessage.classList.add('sent');
+  submitButton.value = 'Sent';
+  submitButton.classList.add('success');
+  submitButton.disabled = true;
+}
+
+function validateEmail() {
+  if (email.value !== email.value.toLowerCase()) {
+    setError(email, 'Invalid email address');
+  } else {
+    setSuccess(email);
+  }
```

Comment:
- [ ] Kindly make sure your email input is validated correctly.
`Hint: After checking if the email is not equal to its lowercase, you may consider preventing the form to be submitted`
![image](https://user-images.githubusercontent.com/97100091/202841488-3cb734c1-a60b-4980-a52d-d0282d16f8d5.png)

Reply:
Fixed.